### ISSUE TITLE
random bots' gameversion and account type doesn't really work

### ISSUE THREAD

**Issue by refringe** [sp-tarkov/server#900]
## **General information**:

**Project issue is for:** </br>
Server

**Expected result**:</br>
PMC bots can be sherpa, bsg crews, EOD. this will be revealed when player killed by PMC bots.

**Actual result**:</br>
All PMC bots are common.

related file:
https://dev.sp-tarkov.com/SPT-AKI/Server/src/branch/master/project/src/generators/BotGenerator.ts
line 427
```
protected getRandomisedGameVersionAndCategory(botInfo: Info): void
    {
        if (botInfo.Nickname === "Nikita")
        {
            botInfo.GameVersion = "edge_of_darkness";
            botInfo.AccountType = MemberCategory.DEVELOPER;

            return;
        }

        const gameVersions = ["standard", "standard", "left_behind", "prepare_for_escape", "edge_of_darkness"];
        const accountTypes = [MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEFAULT,MemberCategory.DEVELOPER, MemberCategory.SHERPA,MemberCategory.SHERPA,MemberCategory.SHERPA,MemberCategory.EMISSARY, MemberCategory.EMISSARY, MemberCategory.EMISSARY]; // 0 = normal, 1 = dev, 256 = sherpa, 512 = emissary
        botInfo.GameVersion = this.randomUtil.getArrayValue(gameVersions);
        botInfo.AccountType = this.randomUtil.getArrayValue(accountTypes);
    }
```
GameVersion and AccountType parameters generated by `getRandomisedGameVersionAndCategory` are not used during generate a bot.
line 138 `generateBot` set some bot info, but no GameVersion and AccountType
line 171 used `getRandomisedGameVersionAndCategory` . but it seems useless

---

[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/51) by _**ghost114514**_

**Comment by refringe** [comment]
Hi, here's `generateBot()` having just run + assigned a bot a different game version:

![image](https://dev.sp-tarkov.com/attachments/c440104e-4d5f-41d4-81b1-be93d2dfca91)

Through our testing we've found it adjusts the color of the players' killers' name in the post-raid menu. Is this not the case in your findings?

---
### Attachments
- [image.png](https://dev.sp-tarkov.com/attachments/c440104e-4d5f-41d4-81b1-be93d2dfca91) (83 KiB)


---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/51#issuecomment-18476) by _**chomp**_

**Comment by refringe** [comment]
it's updated!
a good and fast job!

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/51#issuecomment-18490) by _**ghost114514**_

**Comment by refringe** [comment]
no,it didnt work actually. after edited config/bot.json pmc killed me still have no EOD sign.
i'll edit and build again to examine later

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/51#issuecomment-18531) by _**ghost114514**_

**Comment by refringe** [comment]
Will be fixed in 3.6.0 release

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/51#issuecomment-18568) by _**Kaeno**_

**Comment by refringe** [comment]
3.6.0 released, this should be fixed, closing

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/51#issuecomment-18694) by _**CWX**_
